tutorials/roostats/rs701_BayesianCalculator.py

```diff
@@ -0,0 +1,87 @@
+## \file
+## \ingroup tutorial_roostats
+## \notebook
+## Bayesian calculator: basic exmple
+##
+## \macro_image
+## \macro_output
+## \macro_code
+##
+## \date July 2022
+## \authors Artem Busorgin, Gregory Schott (C++ version)
+
+import ROOT
+
+useBkg = True
+confLevel = 0.90
+
+w = ROOT.RooWorkspace("w", True)
```

Comment:
```suggestion
w = ROOT.RooWorkspace("w")
```
Setting the CINT export to `True` is deprecated for a long time. Can you please also change that in the C++ tutorial? Thanks!